app/assets/javascripts/training/components/Alert.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useState } from 'react';
```

Comment:
You can do import React , {useState} from 'react'. I think that's one of the error in failing build

Reply:
Thank you very much for the suggestion!

Reply:
You can have a look at the other fails too

Reply:
Yes! I am fixing the indentation errors and spacing issues as well. Thanks!